### ISSUE TITLE
[Feature Request] View List of Running Jobs/Backups

### ISSUE THREAD

**Issue by cmidkiff87** [NeoApplications/Neo-Backup#763]
**Is your feature request related to a problem? Please describe it.**
In the case that a Backup fails and never stops, or I accidentally launched a backup that takes forever, I would like the option to view and cancel running backups or jobs 

**Describe the primary solution**
A page to view, manage, and terminate running background jobs (backups, restore, etc.).

**Screenshots**
<details><summary>Mockup</summary>

![image](https://github.com/NeoApplications/Neo-Backup/assets/39914727/75e09b13-612c-486d-a6e2-e918c0b4f26d)

Made using [Moqups](https://app.moqups.com)
</details> 

**System Information (please complete the following information):**

- Device: Google Pixel 5
- Android Version: 13
- ROM: LineageOS 20
- App Version: 8.3.1

**Comment by hg42** [comment]
but you know the notifications?

**Comment by cmidkiff87** [comment]
Yes, but the notification doesn't have any method of killing the task, or otherwise managing the task.

Being able to kill a running backup is the main feature I'm interested in. But I worded it as a task management page, because I feel that that would be a better UI/UX for those purposes

**Comment by hg42** [comment]
there is a "cancel" button for this batch/schedule and a "cancel all"...
